http.go

```diff
@@ -46,7 +46,7 @@ type RasterParams struct {
 	ImageQuality int
 	Token        string
 	Timestamp    time.Time
-	Filename     string
+	File         filecache.File
```

Comment:
`File` is a thing in Go so I wouldn't name this `File` unless it really is an `os.File`